web/beacon-app/cypress/e2e/ProjectDetailPage/ProjectDetailPage.ts

```diff
@@ -286,11 +368,22 @@ And("I click the Create Topic button", () => {
 Then("I should see the new topic in the topic list table", () => {
     cy.get('[data-cy="topicTable"]').within(() => {
         cy.get('tbody>tr>td').eq(0).should('have.text', 'Test-Topic-01')
-        cy.get('tbody>tr>td>span').eq(0).should('have.text', 'active')
-        cy.get('tbody>tr>td').eq(5).should('have.text', '05/12/2023')
+        cy.get('tbody>tr>td>div>span').eq(0).should('have.text', 'Active')
+        cy.get('tbody>tr>td').eq(5).should('have.text', '07/25/2023')
     })
 });
 
+Then("I should create another API key for the rest of the tests", () => {
+    cy.get('[data-cy="addKey"]').should('be.visible').click()
+    cy.get('[data-testid="keyName"]').type('Test Key')
+    cy.get('[data-testid="keyModal"]').within(() => {
+        cy.get('fieldset>div>div>fieldset').eq(0).click()
```

Comment:
I noticed that sometimes when attempting to create a new API key that the following error message would appear: `API Key permissions are required.` The modal has `Full Access` selected by default, so I'm not sure why that error message appeared. To get around it, I clicked full access twice. We should probably create a story to look into this issue.